common/schemamigration.go

```diff
@@ -8,9 +8,10 @@ import (
 )
 
 var (
-	createTableRegex         = regexp.MustCompile(`(?i)create table if not exists ([0-9a-z_]*) *\(`)
-	alterTableAddColumnRegex = regexp.MustCompile(`(?i)alter table ([0-9a-z_]*) add column if not exists ([0-9a-z_]*)`)
-	addIndexRegex            = regexp.MustCompile(`(?i)create (unique )?index if not exists ([0-9a-z_]*) on ([0-9a-z_]*)`)
+	createTableRegex          = regexp.MustCompile(`(?i)create table if not exists ([0-9a-z_]*) *\(`)
+	alterTableAddColumnRegex  = regexp.MustCompile(`(?i)alter table ([0-9a-z_]*) add column if not exists "?([0-9a-z_]*)"?`)
+	addIndexRegex             = regexp.MustCompile(`(?i)create (unique )?index if not exists ([0-9a-z_]*) on ([0-9a-z_]*)`)
+	addNotNullConstraintRegex = regexp.MustCompile(`(?i)alter table ([0-9a-z_]*) alter column "?([0-9a-z_]+)"? set not null`)

```

Comment:
Current regexes assume that column names will always be unquoted; this assumption holds for the current set of migrations but is not generally true. For instance, the `reminders` table contains a field called `when` which is a reserved keyword in Postgres and so must be quoted.

Therefore in both the new regex for `ALTER COLUMN ... SET NOT NULL` queries and the existing `ALTER TABLE ... ADD COLUMN` regex I have added support for (optionally) quoted column names.